proc/process.c

```diff
@@ -474,6 +486,7 @@ int process_load(process_t *process, vm_object_t *o, offs_t base, size_t size, v
 	void *stack;
 	Elf64_Ehdr *ehdr;
 	vm_map_t *map = process->mapp;
+	size_t ustacksz;
```

Comment:
I guess it's a matter of taste where default value is assigned. As an optional change I like the idea of subfunction optionally changing it's value. Just a suggestion
```suggestion
	size_t ustacksz = SIZE_USTACK;
```